connector/src/main/java/org/geysermc/connector/skin/SkinProvider.java

```diff
@@ -412,14 +412,22 @@ private static Skin supplyEars(Skin existingSkin, String earsUrl) {
 
         // if the requested image is a cape
         if (provider != null) {
-            if (image.getWidth() > 64) {
+            if (image.getWidth() > 64 || image.getHeight() > 32) {
```

Comment:
```suggestion
            if (image.getHeight() > 32) {
```

Reply:
But what if the cape size is, say, 72 x 32? I'm honestly not sure how Minecraft handles that, and I can't test with MinecraftCapes since it has a dimension check.

Reply:
My bad I didn't see that it checked both height and width.